pandas/io/excel/_base.py

```diff
@@ -99,12 +102,29 @@
     of dtype conversion.
 engine : str, default None
     If io is not a buffer or path, this must be set to identify io.
-    Supported engines: "xlrd", "openpyxl", "odf", "pyxlsb", default "xlrd".
+    Supported engines: "xlrd", "openpyxl", "odf", "pyxlsb".
     Engine compatibility :
+
     - "xlrd" supports most old/new Excel file formats.
     - "openpyxl" supports newer Excel file formats.
     - "odf" supports OpenDocument file formats (.odf, .ods, .odt).
     - "pyxlsb" supports Binary Excel files.
+
+    .. versionchanged:: 1.2.0
+        The engine xlrd is no longer maintained, and is not supported with
```

Comment:
I *think* need a blank line here to render (make this section the same as in the whatsnew as per formatting)

Reply:
No, for versionchanged this is OK (rst .. ;-))